pkg/cmd/builder/build_test.go

```diff
@@ -213,6 +214,10 @@ func TestParseBuildctlArgsForOCILayout(t *testing.T) {
 		},
 	}
 
+	if runtime.GOOS == "windows" {
+		tests[1].expectedErr = "open D:\\tmp\\oci-layout\\index.json: The system cannot find the path specified."
```

Comment:
D:\\tmp shouldn't be hardcoded?

Reply:
Interesting.
Yeah, I guess it might vary from one install to the other?

I really wish I had a windows install around.

Let me look into this real quick.

Reply:
Ok, that should do the trick.

@AkihiroSuda PTAL.